lib/ts/recipe/emailpassword/types.ts

```diff
@@ -425,13 +390,13 @@ export type FormBaseProps<T> = {
 
     buttonLabel: string;
 
-    error?: string;
-
     validateOnBlur?: boolean;
 
+    clearError: () => void;
+    onError: (error: string) => void;
     onSuccess?: (result: T & { status: "OK" }) => void;
 
-    callAPI: (fields: APIFormField[]) => Promise<FormBaseAPIResponse<T>>;
+    callAPI: (fields: APIFormField[], setValue: (id: string, value: string) => void) => Promise<FormBaseAPIResponse<T>>;
```

Comment:
What is the purpose of `setValue` function?

Reply:
It's used in `emailOrPhoneForm`: we can update the stored value after submitting i.e., guess the international number.